dlinfer/utils/graph/custom_op.py

```diff
@@ -47,7 +48,7 @@ def _impl_abstract_func(*args, **kwargs):
                 return tuple(result)
 
             impl_abstract_func = _impl_abstract_func
-        torch._custom_ops.impl_abstract(qualname)(impl_abstract_func)
```

Comment:
this file is used for graph mode. maca has not supported graph mode.

Reply:
done.